src/dmd/typesem.d

```diff
@@ -3429,40 +3429,16 @@ Expression dotExp(Type mt, Scope* sc, Expression e, Identifier ident, int flag)
             return e;
         }
 
-        Dsymbol searchSym()
-        {
-            int flags = sc.flags & SCOPE.ignoresymbolvisibility ? IgnoreSymbolVisibility : 0;
-
-            Dsymbol sold = void;
-            if (global.params.bug10378 || global.params.check10378)
-            {
-                sold = mt.sym.search(e.loc, ident, flags);
-                if (!global.params.check10378)
-                    return sold;
-            }
-
-            auto s = mt.sym.search(e.loc, ident, flags | IgnorePrivateImports);
-            if (global.params.check10378)
-            {
-                alias snew = s;
-                if (sold !is snew)
-                    Scope.deprecation10378(e.loc, sold, snew);
-                if (global.params.bug10378)
-                    s = sold;
-            }
-            return s;
-        }
-
-        s = searchSym();
+        immutable flags = sc.flags & SCOPE.ignoresymbolvisibility ? IgnoreSymbolVisibility : 0;
+        s = mt.sym.search(e.loc, ident, flags | IgnorePrivateImports);
```

Comment:
This could just be:
```d
s = mt.sym.search(e.loc, ident, (sc.flags & SCOPE.ignoresymbolvisibility) | IgnorePrivateImports);
```

Reply:
I don't think so because `SCOPE.ignoresymbolvisibility` is `0x200` and `IgnorePrivateImports` is `0x80`.  If they were the same value, I think it would work, but as long as they're different, I think this is the way it needs to be done.  Please correct me if I'm wrong.

Reply:
Oh, didn't realize they were different enum 🤦‍♂️